data/payloads/70632d81-bb39-40dd-bfba-cfd1f8196eb6.yml

```diff
@@ -60,6 +60,8 @@ standard_payloads:
   wifi.sh:
     description: Wifi manipulation script
     id: 9f639067-370a-40ba-b7ac-6f1c15d5a158
+    obfuscation_name:
+      - TODO
```

Comment:
can we fill this in with something a bit more explanatory as an example? let's create a few more payload examples while we're at it. also update the documentation page around abilities.

Reply:
TODO is literally the name of the payload :)

Creating more should fall under another ticket - as should updating documentation.  I'm going to close out the current ticket.  We can add a documentation and update payloads ticket to next sprint